src/RunningMode/Webhook.php

```diff
@@ -42,13 +42,6 @@ public function __construct(?Closure $resolveIp = null)
         $this->resolveIp = $resolveIp ?? static fn (): string => $_SERVER['REMOTE_ADDR'];
     }
 
-
-    /**
-     * @param Nutgram $bot
-     * @throws JsonMapper_Exception
-     * @throws InvalidArgumentException
-     * @throws Throwable
-     */
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why some phpdocs got removed?